### ISSUE TITLE
summarize_row_groups should also take `.var` argument in case that "var" is specified

### ISSUE THREAD

**Issue by danielinteractive** [Roche/rtables#66]
There will be many situations where an analyze call to a variable, see below for `AGE`, is preceded by a summary statement. In that case it would be great if the following can work to summarize the number of non-missing variable numbers before:

```r
library(rtables)
DM$AGE[1:50] <- NA

t1 <- basic_table() %>% 
  split_cols_by("ARM") %>%
  split_rows_by("RACE") %>%
  summarize_row_groups(
    label_fstr = "%s (n)", 
    var = "AGE",
    cfun = function(df, ..., .N_col, .var) {
      # c(nrow(df), nrow(df)/.N_col)
      sum(!is.na(df[[.var]])) * c(1, 1/.N_col)
    }) %>%
  analyze("AGE", afun = list_wrap_x(summary) , format = "xx.xx") %>%
  build_table(DM)
t1
```

**Comment by danielinteractive** [comment]
@gmbecker just added this new enhancement request, hope this would not be too difficult

**Comment by gmbecker** [comment]
Keep in mind that content functions currently **must** accept labelstr as a formal argument (`...` is not sufficient). With that context, this is done in ac0c8de189e81bb5dcac1b18dda1abfefa32aed3

**Comment by gmbecker** [comment]
This is addressed for analyze_colvars in 0.3.17.9009. I don't have an eta
currently for summarize_row_groups support.

**Comment by gmbecker** [comment]
Disregard that for now, I'm seeing vignette breakages, you can watch #70 on
the public github repo. when that is closed this will be safe to use.

~G
